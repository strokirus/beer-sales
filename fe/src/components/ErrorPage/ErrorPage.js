import React from 'react';
import {
  Wrapper,
} from '../../cssGlobals/globals';

/**
 * Component to Loading
*/
const ErrorPage = () => (
  <>
    <Wrapper>
      <h1>Something is rotten in the state of Denmark</h1>
      <p>Try contact the administrators of site</p>
    </Wrapper>
  </>
);

ErrorPage.propTypes = {
};

ErrorPage.defaultProps = {
};

export default ErrorPage;
