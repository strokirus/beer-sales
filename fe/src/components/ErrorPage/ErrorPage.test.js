import React from 'react';
import renderer from 'react-test-renderer';
import ErrorPage from './ErrorPage';

describe('(Component) ErrorPage', () => {
  it('should render an ErrorPage', () => {
    const tree = renderer.create(<ErrorPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
