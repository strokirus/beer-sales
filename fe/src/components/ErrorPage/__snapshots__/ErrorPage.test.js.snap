// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) ErrorPage should render an ErrorPage 1`] = `
<section
  className="sc-AxhCb jmvNtO"
>
  <h1>
    Something is rotten in the state of Denmark
  </h1>
  <p>
    Try contact the administrators of site
  </p>
</section>
`;
